schemas/definition.json

```diff
@@ -44,11 +68,54 @@
         }
       }
     },
+    "files": {
+      "type": "array",
+      "items": {
+        "type": "object",
+        "required": [
+          "path"
+        ],
+        "additionalProperties": false,
+        "properties": {
+          "path": {
+            "type": "string",
+            "description": "The absolute path of the file as found in the component."
+          },
+          "facet": {
```

Comment:
`facet` and `facets`?  why both?  Seems like just facets. It might have been me who added both. either way it is wrong.